README.md

```diff
@@ -27,6 +27,6 @@ If you want to preview this lesson locally, you will need the [{sandpaper} R pac
 [porter-nathaniel]: https://carpentries.org/instructors/#ndporter
 [wheeler_jon]: https://carpentries.org/instructors/#jonathanwheeler01
 [word_karen]: https://carpentries.org/team/
```

Comment:
We should probably remove this and any other links to past maintainers to make the code cleaner.

Reply:
💯 but also the links to us do not go to individuals anymore... should we replace them? (see original comment for ideas)

Reply:
Yes, I think linking to our github profiles seems the most robust with the least work.